pkg/resources/ali/ali.go

```diff
@@ -60,7 +60,7 @@ type clusterDiscoveryALI struct {
 
 func ListResourcesALI(aliCloud aliup.ALICloud, clusterName string, region string) (map[string]*resources.Resource, error) {
 	if region == "" {
-		region = aliCloud.Region()
+		aliCloud.Region()
```

Comment:
This seems nonsensical. If `region` is not used, then this whole if block should be removed.

Reply:
so...I will remove them together.